src/CMakeLists.txt

```diff
@@ -299,6 +299,8 @@ set(SOURCE_FILES
 	framework/stdext/qrcodegen.cpp
 	framework/util/color.cpp
 	framework/util/crypt.cpp
+	framework/proxy/proxy.cpp
```

Comment:
@mehah
Why did you move it to the end? Aren't `CMakeLists.txt` files ordered alphabetically?

Reply:
@gesior 
This fix the build with unity